daemon/cluster/executor/container/executor.go

```diff
@@ -136,23 +136,34 @@ func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
 }
 
 func (e *executor) Configure(ctx context.Context, node *api.Node) error {
-	na := node.Attachment
```

Comment:
I'm not too familiar with this, but is `node.Attachment` no longer needed? (is this now included in `node.LbAttachments` ?)

Reply:
Correct.  node.Attachment has been deprecated https://github.com/docker/swarmkit/blob/master/api/objects.proto#L64